deploy/docker/entrypoint.sh

```diff
@@ -21,6 +21,20 @@ if [[ -n ${APPSMITH_SEGMENT_CE_KEY-} ]]; then
     || true
 fi
 
+if [[ -n "$FILESTORE_IP_ADDRESS" ]]; then
```

Comment:
Can we try running this container once? I think this will fail to start the container. Since we now have a `set -e` above this, if this variable isn't defined, the entrypoint script will immediately break.

```suggestion
if [[ -n "${FILESTORE_IP_ADDRESS-}" ]]; then
```

Let's also run this once, on non-CloudRun systems to ensure we aren't missing anything else here please.